apps/charterafrica/src/components/YoutubeVideoPlayer/YoutubeVideoPlayer.js

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable jsx-a11y/media-has-caption */
+import { Box, styled } from "@mui/material";
+import PropTypes from "prop-types";
+import React, { useEffect, useRef } from "react";
+import videojs from "video.js";
+import "videojs-youtube";
+import "video.js/dist/video-js.css";
+
+const StyledDiv = styled(Box)({
+  minWidth: "100%",
+  height: "100%",
+  "& .video-js .vjs-big-play-button": {
+    height: "100%",
+    left: 0,
+    top: 0,
+    width: "100%",
+    borderRadius: 0,
+    background: "rgba(62,32,44, 0.9)",
+  },
+  "& .video-js .vjs-big-play-button:hover": {
+    height: "100%",
+    left: 0,
+    top: 0,
+    width: "100%",
+    borderRadius: 0,
+    background: "rgba(62,32,44, 0.8)",
+  },
+  ".vjs-button > .vjs-icon-placeholder:before, .video-js .vjs-big-play-button .vjs-icon-placeholder:before":
+    {
+      position: "relative",
+      display: "flex",
+      justifyContent: "center",
+      alignItems: "center",
+      fontSize: 70,
+    },
+});
+
+function YoutubeVideoPlayer({ videoId, width }) {
+  const videoRef = useRef(null);
+  const playerRef = useRef(null);
+  const parentRef = useRef(null);
+
+  const src = `https://www.youtube.com/watch?v=${videoId}`;
+
+  useEffect(() => {
+    if (videoRef.current && !playerRef.current) {
+      const options = {
+        autoplay: false,
+        controls: true,
+        preload: "auto",
+        sources: [{ src, type: "video/youtube" }],
+        techOrder: ["youtube"],
+        height: "auto",
+        width,
```

Comment:
Any reason why we're going for fixed with instead of [`fluid`](https://docs.videojs.com/player#fluid)?

Reply:
Yes.

100% width didn't work with `height: auto`.

We end up with something like

<img width="1179" alt="image" src="https://user-images.githubusercontent.com/39160236/225239987-b6ba8764-56f2-44fc-b5b2-a507b7bd179c.png">

So I thought it would be best if width is supplied as a prop.

Reply:
That sounds like a `fill` mode. In `fluid` mode, all you need is an `aspectRatio`, no?

https://videojs.com/guides/layout/

Reply:
@koechkevin ?